### PR TITLE
Add navigation.emulate_link_click

### LINKED ISSUE #88: Navigation click_link(target="") method?
I would like to be able to emulate clicking a navigation menu link from code. The undocumented method 'open_form' gets me most of the way there but it doesn't show the menu link as selected.

### PR DESCRIPTION
**Pull request by hugetim**
Closes #88

### CODE REVIEW

**Comment by hugetim** [comment]
Looks like I need to bite the bullet and get black installed. There's also probably a better name for this function.

**Comment by s-cork** [comment]
This might work for you. 

```
pip install pre-commit
pre-commit install
pre-commit run --all-files
```

File: client_code/navigation.py [hunk 1]
```diff
@@ -46,6 +46,12 @@ def open_form(form_name, full_width=False):
     get_open_form().content_panel.clear()
     get_open_form().content_panel.add_component(form, full_width_row=full_width)
 
+    
+def emulate_link_click(target):
+    """Emulate clicking a menu link"""
+    matching_links = [link for link in _links if link.tag.target == target]
+    matching_links[0].raise_event('click')
+
```
Comment (s-cork) [inline client_code/navigation.py:54]: A nicer error when there are no matching links. 

You can also do a loop and break out when you find the first one. And use the no break syntax. 

```python
for link in links:
    if condition:
        break
else: # no break
    raise ...
link.raise_event('click')
```

Re name of the function. `go_to`or `navigate_to`?

**Review by s-cork — APPROVED** [review]
@meatballs  - LGTM - over to you.

File: client_code/navigation.py (hunk 1 above)
Comment (hugetim) [reply client_code/navigation.py]: I implemented this, changed name to go_to, and also tried adding a more helpful error message to the get_form function.

**Comment by hugetim** [comment]
Ok, I'm stopping with the error message fiddling now.

**Comment by meatballs** [comment]
Could we have some docs and a changelog entry?

**Pull request merged**